front/src/StandardButton.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import SRRButton from "./StandardButton";
+import renderer from "react-test-renderer";
+
+const testFunc = () => {
+  return "Yeah, that'll do.";
+};
+
+describe("Button can be clicked", () => {
+  it("Test Button with Snapshot", () => {
+    const Button = renderer.create(
+      <SRRButton
+        buttonText={<p>hurrdurr</p>}
+        onClick={testFunc}
+        enabled={true}
+      />
+    );
+    let but = Button.toJSON();
+    expect(but).toMatchSnapshot();
+    const value = but.props.onClick();
```

Comment:
what are you testing here? the fact that the button can be clicked?

Reply:
yes